### DISCUSSION TITLE
Any plan to provide aarch64 ZFS package for CentOS/RHEL8 ?

### DISCUSSION THREAD

**Discussion by xin3liang** [openzfs/zfs#12414] [General]
Hi,  I tried to install ZFS on aarch64 CentOS 8.4 following the **[getting start guide](https://openzfs.github.io/openzfs-docs/Getting%20Started/RHEL%20and%20CentOS.html#)**. But it turns out that there is no ZFS package for aarch64 CentOS 8 yet :-(.

Do you have any plan to release aarch64 ZFS packages for aarch64 CentOS/RHEL8? 
Or anything we (Linaro) can help to release aarch64 CentOS/RHEL8 ZFS packages? What are the resources missing to release aarch64 packages? We are really very happy to help :-).

**Comment by tonyhutter** [comment]
Sorry, we have no plans to build packages for aarch64 CentOS 8.4.
